Lesson 2 HW Ernest.playground/Contents.swift

```diff
@@ -0,0 +1,49 @@
+import Darwin
+let number1 = 2.5
+let number2 = 9.7
+let number3 = 6.9
+let number4 = 8.2
+let wholeNumber1 = Int(floor(number1))
+let fractionalNumber1 = number1.truncatingRemainder(dividingBy: 1)
+let wholeNumber2 = Int(floor(number2))
+let fractionalNumber2 = number2.truncatingRemainder(dividingBy: 1)
+let wholeNumber3 = Int(floor(number3))
+let fractionalNumber3 = number3.truncatingRemainder(dividingBy: 1)
+let wholeNumber4 = Int(floor(number4))
```

Comment:
You can drop `floor` and leave only `Int` here because casting to `Int` will already remove the fractional part without rounding a number with mathematical rounding rule